pkg/proxy/userspace/proxier_test.go

```diff
@@ -894,7 +914,17 @@ func TestProxyUpdatePortal(t *testing.T) {
 		}}},
 	}
 	p.OnServiceUpdate(svcv0, svcv1)
-	_, exists := p.getServiceInfo(service)
+
+	// Wait until the proxy updates the service; fail if it doesn't do so
+	// before the timeout is reached
```

Comment:
why can't this use `waitForServiceInfo`?

Reply:
@danwinship this one is actually waiting for a serviceInfo that already exists to go away, so it can't use waitForServiceInfo(). I've updated the comment to make that clear.